app/views/galleries/show.rb

```diff
@@ -15,6 +15,19 @@ def js_vars
       [{ name: 'pageName', value: 'collections/galleries' }]
     end
 
+    def head_meta
+      description = @gallery.description
+      description = description.strip! || description
```

Comment:
no need to strip it and re-assign, just do .strip  on the line above no need for !